src/main.rs

```diff
@@ -8,17 +10,15 @@ use rocket::{
 use rocket_dyn_templates::Template;
 use ssi::jwk::{Params, JWK};
 
-use oidc4vci_issuer::*;
-
 #[rocket::post("/token", data = "<query>")]
 fn post_token_default_op_state(
-    query: rocket::form::Form<token::TokenQueryParams>,
+    query: rocket::form::Form<TokenRequest>,
```

Comment:
If the intention is for this to be a generic on-spec oid4vci issuer, I wonder whether we should leave this as TokenQueryParams and just define TokenRequest in the microservice?

Reply:
Agreed. And in the future oid4ci-rs will define a struct with additional properties generics but for now just copy paste TokenRequest into the microservice

Reply:
Ok, will do